plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -235,6 +238,7 @@ private List<DeltaLakeSplit> splitsForFile(
                     fileSize,
                     addFileEntry.getModificationTime(),
                     ImmutableList.of(),
+                    SplitWeight.fromProportion(Math.min(Math.max((double) fileSize, minimumAssignedSplitWeight), 1.0)),
```

Comment:
This is always going to be `1.0`. I think for non-splittable queries it's okay to just always set this as `SplitWeight.standard()`. This is only used for update/delete queries anyway